solana/solana-ibc/programs/solana-ibc/src/execution_context.rs

```diff
@@ -135,7 +141,8 @@ impl ibc::ExecutionContext for IbcStorage<'_, '_> {
         msg!("store_connection({}, {:?})", path, connection_end);
         let connection = trie_ids::ConnectionIdx::try_from(&path.0)?;
         let serialised = storage::Serialised::new(&connection_end)?;
-        let hash = serialised.digest();
```

Comment:
We might want to get rid of `Serialised::digest` and `Serialised::digest_with_client` methods since they are now unused and potentially confusing.

Reply:
sounds good